Druid/DRGreaterConstraint.class.st

```diff
@@ -30,6 +30,16 @@ DRGreaterConstraint >> isIncludedIn: otherConstraint [
 	^ otherConstraint hasAllValuesGreaterThan: constantValue
 ]
 
+{ #category : 'reversing' }
+DRGreaterConstraint >> minABCDDistanceToBeSatisfiable [
+
+	"TODO
+	I'm not sure, but I think returning 1 here is fine, and may allow it to optimize more cases.
+	Needs testing though"
+	"^ 1"
+	^ nil
+]
```

Comment:
Should we open an issue about the current support of the constraints and what is missing?

Reply:
Maybe we should. I leave these TODO to grep later :p but an issue is probably better, I'll do that later